t/daemon/formats/jats.xml

```diff
@@ -54,7 +54,7 @@
   </front>
   <body>
     <sec id="S1">
-      <title>Introduction</title>
+      <title>1. Introduction</title>
```

Comment:
The one question I'd have here is whether there is a way to wrap some ephemeral element around the numbers, also in JATS. I have found that very useful in the HTML output as for certain NLP tasks being able to reliably discard the numbering makes life very convenient.

It's not the hardest regex to write by hand if you were using this JATS form, so just something to remark in passing.

Reply:
Ah, I opened the PRs in inverse order, just saw that you've raised the same point in the other one. Apologies, feel free to disregard.